src/Humanizer.Tests/project.json

```diff
@@ -1,12 +1,12 @@
 ﻿{
-    "dependencies": {
-        "ApiApprover": "3.0.1",
-        "ApprovalTests": "3.0.10",
-        "ApprovalUtilities": "3.0.10",
-        "Mono.Cecil": "0.9.6.1",
-        "xunit": "2.1.0",
-        "xunit.runner.visualstudio": "2.1.0"
-    },
+  "dependencies": {
+    "ApiApprover": "3.0.1",
+    "ApprovalTests": "3.0.10",
+    "ApprovalUtilities": "3.0.10",
+    "Mono.Cecil": "0.9.6.1",
+    "xunit": "2.1.0",
+    "xunit.runner.visualstudio": "2.1.0"
+  },
```

Comment:
@rdoubleui can you revert the changes to this file, they are irrelevant to the feature, right?